src/Avalonia.Controls/SplitView/SplitViewTemplateSettings.cs

```diff
@@ -17,12 +17,14 @@ public class SplitViewTemplateSettings : AvaloniaObject
             AvaloniaProperty.Register<SplitViewTemplateSettings, GridLength>(
                 nameof(PaneColumnGridLength));
 
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("AvaloniaProperty", "AVP1032", Justification = "This property is supposed to be a styled readonly property.")]
```

Comment:
I'm not familiar with this class, but does it need to be a styled property given that afaics this is only used for communicating with the template and can't actually be styled?

Reply:
In this PR I didn't want to change any property types. Even though I believe we still need a readonly styled properties (or inheritable direct properties, which is less likely), this one probably can be a direct as it seems.